app/src/main/res/drawable/toolbar_bg.xml

```diff
@@ -2,7 +2,7 @@
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
 
     <stroke
-        android:width="1dp"
+        android:width="0dp"
         android:color="@color/veryLightPink" />
```

Comment:
Make a seperate file for this use that instead as this file being for toolbar as well

Reply:
ok sure

Reply:
undo this change

Reply:
which change that 0 dp change or new file change?

Reply:
obviously this file

Reply:
Ok I have forked new repo as my previous repo has some issues in the building. I am making a branch from that repo